packages/jest-haste-map/src/index.ts

```diff
@@ -879,9 +892,6 @@ class HasteMap extends EventEmitter {
             return null;
           };
 
-          const relativeFilePath = fastPath.relative(rootDir, filePath);
```

Comment:
is it safe to extract from here? this is async, in theory its values could have changes since before waiting for `changeQueue`, no?

Reply:
`rootDir` and `filePath` are closed over, so they shouldn't change, but `fileMetadata` may technically change in the mean time, so I'll bring it back there